system-addon/common/Reducers.jsm

```diff
@@ -208,7 +208,7 @@ function Sections(prevState = INITIAL_STATE.Sections, action) {
           // This can be overridden if initialized is defined in the action.data
           const initialized = action.data.rows ? {initialized: true} : {};
 
-          if (action.data.rows && section.rows.find(card => card.pinned)) {
+          if (action.data.rows && action.data.rows.length > 0 && section.rows.find(card => card.pinned)) {
             // Make sure that pinned cards stay at their current position
```

Comment:
Can you just update this comment to reflect the change you just made?

Reply:
OK, thanks, updated.